ui/values/localizable/es.go

```diff
@@ -12,7 +12,7 @@ const ES = `
 "receive" = "Recibir";
 "online" = "En línea, ";
 "offline" = "Desconectado, ";
-"showDetails" = "Mostrar detalles";
+"viewDetails" = "View details";
```

Comment:
This should be translated to spanish

Reply:
it doesn't necessarily have to, we would have a translator do all non-english translations

Reply:
The idea is to have a place holder for the other languages pending when we do a general sting localization like the #906